MigrationV3V4/CSharp/ContosoHelpdeskChatBot-V4NetFramework/ContosoHelpdeskData/ContosoHelpdeskData.sqlproj

```diff
@@ -0,0 +1,78 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we even need this for the v4 bot? Aren't we just mocking the call to the database?

Reply:
Removed the .sqlproj